src/main/java/com/w6/nlp/DateTimeParser.java

```diff
@@ -0,0 +1,28 @@
+package com.w6.nlp;
+
+import edu.stanford.nlp.simple.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+public class DateTimeParser {
+
+    @NotNull
+    public static List<String> parseDateAndDateFromString(String text) {
```

Comment:
ParseDateAndTime
FromString is obvious from parameters.
And not @notNull comment was about parameter.
@notNull String text